libs/access-control/Cargo.toml

```diff
@@ -8,7 +8,11 @@ actix-http.workspace = true
 app-error.workspace = true
 anyhow.workspace = true
 async-trait.workspace = true
-casbin = { version = "2.2.0", features = ["cached", "runtime-tokio", "incremental"] }
+casbin = { version = "2.2.0", features = [
```

Comment:
we can try to use 2.5.0 version

Reply:
Just tried using the 2.5.0 version - there are some changes in function signature. Will do the upgrade in another PR instead and keep 2.2.0 for this PR.